### ISSUE TITLE
Periodic table issue

### ISSUE THREAD

**Issue by Crispae** [BeckResearchLab/small-molecule-design-toolkit#2]

**Comment by dacb** [comment]
Can you provide more description?

**Comment by Crispae** [comment]
This issue has been resolved by  running  this in anaconda environment ,but there are some other issue  regarding sk learn which  are deprecated that is  **sklearn.metrics.jaccard_similarity_score** is updated  to **sklearn.metrics.jaccard_score**.

Update this **from sklearn.preprocessing import  Imputer** in classification.py  to **from sklearn.impute import SimpleImputer**
as it is updated in new version of sklearn.

The use of axis in **Imputer(missing_values='NaN', strategy='median', axis=0)**  is also removed from this i think they make it default .
